src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Services/OpenApiDocumentServiceTestsBase.cs

```diff
@@ -229,7 +229,7 @@ public ControllerActionDescriptor CreateActionDescriptor(string methodName = nul
 
         action.AttributeRouteInfo = new()
         {
-            Template = action.MethodInfo.GetCustomAttribute<RouteAttribute>()?.Template,
+            Template = action.MethodInfo.GetCustomAttribute<RouteAttribute>()?.Template ?? string.Empty,
```

Comment:
This change is to make this assertion happy:

https://github.com/dotnet/aspnetcore/blob/02125bdb3eced4a8a562181680581099296b7f72/src/OpenApi/src/Extensions/ApiDescriptionExtensions.cs#L44